src/fluree/db/ledger/json_ld.cljc

```diff
@@ -90,6 +94,31 @@
         db*   (or db (ledger-proto/-db ledger (:branch opts*)))]
     (jld-commit/commit ledger db* opts*)))
 
+(defn default-context
+  "Returns default context at t (must be internal negative value)"
+  [ledger t]
+  (go-try
+    (let [latest-db (db ledger nil)
+          t*        (cond
+                      (string? t) (<? (time-travel/datetime->t latest-db t))
+                      (pos-int? t) (- t)
+                      (neg-int? t) t)
+          db-conn   (:conn latest-db)
+          flakes-ch (query-range/time-range latest-db :tspo = []
```

Comment:
I think if you make this a `:psot` query you can just fetch the precise flake you care about and avoid having to parse:
```clojure
(query-range/time-range latest-db :psot = [const/$_ledger:context] {:from-t t* :to-t t*})
```

Reply:
Now that I think about it more, you could get even more precise with a regular `index-range` call, since the `sid` of a commit is the `t`.
```clojure
(let [default-context-sid (->> (<? (index-range latest-db :spot = [t* const/$_ledger:context] {:flake-limit 1}))
                               (first)
                               (flake/o))
      address (->> (<? (index-range latest-db :spot = [default-context-sid const/$_address] {:flake-limit 1}))
                   (first)
                   (flake/o))])
```
This allows you to skip reifying the commit entirely and just call `load-default-context` on the address.

Reply:
Yep, that worked! Thanks!